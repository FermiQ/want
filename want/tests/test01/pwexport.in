&inputpp
  prefix='silicon'
  outdir='./'
  pseudo_dir='./'
  psfile(1)='Si.vbc.UPF'
  single_file= .FALSE.
/
