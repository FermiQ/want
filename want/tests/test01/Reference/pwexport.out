
     Program POST-PROC v.2.1.5  starts ...
     Today is 17Oct2005 at 17:31:58 

     Reading file silicon.save ... 
     read complete

     Reading file silicon.save ... 
     read complete

     nbndx  =    16  nbnd   =    16  natomwfc =     8  npwx   =     869
     nelec  =    8.00 nkb   =     8  ngl    =     117
